src/prolong_restrict/pr_loops.hpp

```diff
@@ -17,8 +17,12 @@
 // so.
 //========================================================================================
 
-#ifndef MESH_MESH_REFINEMENT_LOOPS_HPP_
-#define MESH_MESH_REFINEMENT_LOOPS_HPP_
+#ifndef PROLONG_RESTRICT_PR_LOOPS_HPP_
+#define PROLONG_RESTRICT_PR_LOOPS_HPP_
+
+// TODO(JMM): Is the "mesh" directory where prolongation/restriction
+// machinery belongs? Or should we have a separate directory for this?
+// There's also the AMR criteria in the src/refinement directory
```

Comment:
I don't quite understand this comment. Isn't this PR moving prolongation and restriction into their own directory?

Reply:
Git might have screwed something up when I merged develop in. As you say this comment shouldn't be here.

Reply:
Removed.